controller/login.go

```diff
@@ -157,11 +202,19 @@ func (c *LoginController) Refresh(ctx *app.RefreshLoginContext) error {
 	if err != nil {
 		log.Error(ctx, map[string]interface{}{
 			"err": err,
-		}, "Unable to get Keycloak token endpoint URL")
+		}, "unable to get Keycloak token endpoint URL")
 		return jsonapi.JSONErrorResponse(ctx, errors.NewInternalError(ctx, errs.Wrap(err, "unable to get Keycloak token endpoint URL")))
 	}
 
-	rpt, err := auth.GetEntitlement(ctx, entitlementEndpoint, nil, *token.AccessToken)
+	resources, err := c.getEntitlementResourceRequestPayload(ctx, token.AccessToken)
+	if err != nil {
+		log.Error(ctx, map[string]interface{}{
+			"err": err,
+		}, "failed to obtain create entitlement resource request ")
+		return jsonapi.JSONErrorResponse(ctx, goa.ErrInternal(err.Error()))
+	}
+
+	rpt, err := auth.GetEntitlement(ctx, entitlementEndpoint, resources, *token.AccessToken)
```

Comment:
Yes. But since we will most likely use something else soon let's just obtain a regular access token if there is any problem with obtaining entitlement.